docs/planning/bug_triage.md

```diff
@@ -0,0 +1,44 @@
+# Knative Eventing Bug Triage and Management
+
+This document summarizes how bugs will be triaged and managed in Knative Eventing WG.
+This is a supplement to [Sprint planning](sprintplanning.md) process.
+
+## Goal
+
+- Keep a low bug count. Bug count should not be consistently increasing release over release esp [priority/important-longterm](#Bug-priorities).
+- Process shouldn't be taxing for a single person or the team.
+- Maintain high product quality.
+
+## Bug priorities
+
+- priority/critical-urgent: Bugs that need to be fixed now, and possibly cherrypicked into the most recent release. May need patch-fix.
```

Comment:
maybe include the wording `regression` something that use to work now it doesn't work, meaning a potential test work is need it to close the gap.

Reply:
all regressions may not necessarily belong to this bucket. If we are ready to delay the features committed in the current release and fix a bug then the bug qualifies as critical-urgent.